mava/utils/sebulba_tools.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+
+def configure_computation_environment() -> None:
+    """Configure the computation environment for JAX.
+
+    Note: Must be called before any JAX computation is run.
+    """
+    os.environ["JAX_USE_PJRT_C_API_ON_TPU"] = ""
+
```

Comment:
JAX_USE_PJRT_C_API_ON_TPU no longer has an effect (i.e. it always defaults to true).
```suggestion
```